src/components/FilterBarInline/FilterBarInline.mdx

```diff
@@ -347,15 +354,15 @@ const renderMultiSelect = ({ isBlockElement, multiSelectWidth }) => {
 <Source
   code={`
 // function for renderChildren prop of FilterBar 
-const renderFilterComponents = ({ layout, width }) => {
+const renderFilterComponents = ({ closeOnBlur, layout, width }) => {
   return (
     <>
       <MultiSelectGroup
         id="multiselect-group"
         labelText="MultiSelect Group"
         layout={layout}
         multiSelectWidth={width}
-        renderMultiSelect={renderMultiSelect}
+        renderMultiSelect={renderMultiSelect.bind(this, closeOnBlur)}
```

Comment:
Since `closeOnBlur` is not an argument for `renderMultiSelect` or a prop in `MultiSelectGroup`, this is required to pass it to nested `MultiSelect` components.

Reply:
Ah gotcha. Maybe another step is to also allow it in `MultiSelectGroup`? Seems a bit of a rabbit hole for this prop value to go down to `MultiSelect`.